src/components/modals/GameRunningModal.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div id="gameRunningModal" :class="['modal', {'is-active': isOpen}]">
+        <div class="modal-background" @click="close"></div>
+        <div class="modal-content">
+            <div class='notification is-info'>
+                <h3 class="title" v-if="isSteamGame">{{ activeGame.displayName }} is launching via Steam</h3>
+                <h3 class="title" v-else>{{ activeGame.displayName }} is starting</h3>
+                <h5 class="title is-5">Close this message to continue modding.</h5>
+                <div v-if="isSteamGame">
+                    <p>If this is taking a while, it's likely due to Steam starting.</p>
+                    <p>Please be patient, and have fun!</p>
+                </div>
+            </div>
+        </div>
+        <button class="modal-close is-large" aria-label="close" @click="close"></button>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import GameManager from "../../model/game/GameManager";
+import { StorePlatform } from "../../model/game/StorePlatform";
+
+@Component
+export default class GameRunningModal extends Vue {
+    activeGame = GameManager.activeGame;
+    isSteamGame = this.activeGame.activePlatform.storePlatform === StorePlatform.STEAM;
+
+    close() {
+        this.$store.commit('closeModals');
```

Comment:
Nit but can we rename this event to `closeGameRunningModal`
Feels like we shouldn't be closing all in one go and instead should close whichever has the priority overlay. (Thinking errors vs game running modals sharing the same event)

Reply:
I agree, on the hindsight closing all modals at once doesn't sound that appealing. Renamed the commit name.